shared/js/ui/templates/privacy-practices.es6.js

```diff
@@ -17,11 +17,11 @@ module.exports = function () {
     <div class="privacy-practices site-info site-info--full-height card">
         <div class="js-privacy-practices-hero">
             ${hero({
-            status: tosdrStatus,
-            title: domain,
-            subtitle: `${tosdrMsg} Privacy Practices`,
-            showClose: true
-            })}
+        status: tosdrStatus,
+        title: domain,
+        subtitle: `${tosdrMsg} Privacy Practices`,
+        showClose: true
+    })}
```

Comment:
Re: all the whitespace changes in the templates, I tweaked a few of them manually in #211. I then ran `--fix` here, and Eslint reverted them to look like this. Ah well.